sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceClientTestBase.java

```diff
@@ -15,6 +15,8 @@
 public abstract class TableServiceClientTestBase extends TestProxyTestBase {
     protected static final HttpClient DEFAULT_HTTP_CLIENT = HttpClient.createDefault();
 
+    protected static boolean usingEntraAuth;
```

Comment:
Is this value ever used?

Reply:
@vcolin7 It's a left-in shard from when figuring out how to manage the tests that need connection strings. I'll remove this.